pkg/jsonSchemaValidator/validator.go

```diff
@@ -0,0 +1,27 @@
+package jsonSchemaValidator
+
+import (
+	"github.com/ghodss/yaml"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+type JsonSchemaValidator struct {
```

Comment:
rename everything jsonSchemaValidator to yamlSchemaValidator
you use yamls as an input and that is what you're checking, json schema validation is an internal implementation